.github/workflows/audit.yml

```
@@ -42,4 +42,4 @@ jobs:
       - name: Install cargo-audit
         run: cargo install cargo-audit
       - name: Run audit check
-        run: cargo audit
+        run: cargo audit --ignore RUSTSEC-2024-0370
```

I think it is good to leave a comment about rationale too:
```suggestion
        # Ignored until https://github.com/apache/datafusion/issues/15571
        run: cargo audit --ignore RUSTSEC-2024-0370
```